.drone.yml

```diff
@@ -35,14 +35,23 @@ pipeline:
   #     branch: master
 
   publish:
-    image: plugins/docker
-    repo: drone/drone
-    username: ${DOCKER_USERNAME}
-    password: ${DOCKER_PASSWORD}
-    tag: [ 0.6 ]
-    when:
-      branch: master
-      event: push
+
+    docker:
```

Comment:
What do you mean? I set the publish step indentation to one level higher than the key. The indentation on the file looks weird to me in the first-place, but I didn't want to completely re-format the file, because that's a great way of being a pain and getting changes rejected.